downstream/modules/builder/con-additional-build-files.adoc

```diff
@@ -2,16 +2,16 @@
 
 = Additional build files
 
-You can add any file to the build context directory by referring or copying them to the `additional_build_steps` section of the definition file. The format is a list of dictionary values, each with a `src` and `dest` key and value.
+You can add any external file to the build context directory by referring or copying them to the `additional_build_steps` section of the definition file. The format is a list of dictionary values, each with a `src` and `dest` key and value.
 
-Each list item must be a dictionary containing the following (non-optional) keys:
+Each list item must be a dictionary containing the following required keys:
 
-.`src`
-Specifies the source files to copy into the build context directory. This can be an absolute path (for example, `/home/user/.ansible.cfg`), or a path that is relative to the execution environment file. Relative paths can be glob expressions matching one or more files (for example,  `files/*.cfg`).
+== `src`
```

Comment:
This should not be a subheading here but rather a list under the statement "Each list item must be a dictionary containing the following required keys:"

I would either change back to the .`scr` and .`dest` block title format or make it a description list. See 
https://asciidoctor.org/docs/asciidoc-writers-guide/#block-titles
https://asciidoctor.org/docs/asciidoc-writers-guide/#description-lists for more info.